extension/src/NotebookListener.js

```diff
@@ -0,0 +1,74 @@
+import { NotebookActions } from '@jupyterlab/notebook';
```

Comment:
The purpose of this file is to serve as a listener to all relevant notebook actions that might concern our monitor so we know which cell is being run and which monitor to fire up. If there's a clear problem in this file or a better way to do this, I would love feedback here